package.json

```diff
@@ -162,13 +162,15 @@
     "react": ">=17.0.2",
     "react-dom": ">=17.0.2",
     "react-redux": "^7.2.5",
+    "react-router-dom": "^5.3.3",
+    "react-router-dom-v5-compat": "^6.3.0",
```

Comment:
Oh no, a lib to make another lib compatible?!? Probably no other way but lets hope all these downgrades get fixed in the near future.

Reply:
Considering it's the respected authors of react-router that suggest this "route" I'm ok with it :grin: